setup.py

```diff
@@ -0,0 +1,83 @@
+#!usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) Pyronear contributors.
+# This file is dual licensed under the terms of the CeCILL-2.1 and AGPLv3 licenses.
+# See the LICENSE file in the root of this repository for complete details.
+
+"""
+Package installation setup
+"""
+
+import os
+import subprocess
+from setuptools import setup, find_packages
+
+
+package_name = 'pyronearEngine'
+with open(os.path.join('pyronearEngine', 'version.py')) as version_file:
+    version = version_file.read().strip()
+sha = 'Unknown'
+
+cwd = os.path.dirname(os.path.abspath(__file__))
+
+try:
+    sha = subprocess.check_output(['git', 'rev-parse', 'HEAD'], cwd=cwd).decode('ascii').strip()
+except Exception:
+    pass
+
+if os.getenv('BUILD_VERSION'):
+    version = os.getenv('BUILD_VERSION')
+elif sha != 'Unknown':
+    version += '+' + sha[:7]
+print("Building wheel {}-{}".format(package_name, version))
+
+
+with open('README.md') as f:
+    readme = f.read()
+
+with open('requirements.txt') as f:
+    requirements = f.read().splitlines()
+
+setup(
+    # Metadata
+    name=package_name,
+    version=version,
+    author='PyroNear Contributors',
+    author_email='pyronear.d4g@gmail.com',
+    maintainer='Pyronear',
+    description='Pyronear Engine is a repository that aims to deploy pyronear',
```

Comment:
small typo here: "that aims at deploying" ;)

Reply:
Thanks :)